### PR TITLE
Add mongodb plugins

### PR DESCRIPTION
**Pull request by NoxInmortus**
Add 3 plugins :
- mongodb_conn : Graph mongodb connections metrics (compatible mongo v4)
- mongodb_cpu : Graph cpu used by mongod
- mongodb_ram : Graph memory used by mongod

### CODE REVIEW

File: plugins/mongodb/mongodb_ram [hunk 1]
```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/perl -w
+# -*- mode: cperl; mode: autopair -*-
+# Magic markers:
+#%# family=auto
+#%# capabilities=autoconf
+# mongodb_memory --- Munin plugin for monitoring mongodb memory usage.
```
Comment (sumpfralle) [inline plugins/mongodb/mongodb_ram]: Probably you can remove this comment section, too?

File: plugins/mongodb/mongodb_conn [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+def run():
+    connections = _get_connections();
+    for connection, value in connections.items():
+        print connection + ".value", value
+
+def config():
+    print """
+graph_title MongoDB Connections Count
+graph_vlabel Connections count
+graph_category mongodb
```
Comment (sumpfralle) [inline plugins/mongodb/mongodb_conn]: Please use the category `db` (same as the other mongodb plugins).

File: plugins/mongodb/mongodb_conn [hunk 3]
```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+"""
+    MongoDB Connection Count Plugin
```
[the remark this answers is not in the document]
Comment (NoxInmortus) [reply plugins/mongodb/mongodb_conn]: I did not get this one, using perldoc header format for python script ?

[the remark this answers is not in the document]
Comment (sumpfralle) [reply plugins/mongodb/mongodb_conn]: Yes :)

You could take a look at the [smart_ plugin](https://github.com/munin-monitoring/munin/blob/master/plugins/node.d/smart_) as an example.

[the remark this answers is not in the document]
Comment (NoxInmortus) [reply plugins/mongodb/mongodb_conn]: Should be okay now

**Comment by sumpfralle** [comment]
I squashed your commits into one and merged it.

Thank you!

**Pull request merged**